projects/igniteui-angular/src/lib/expansion-panel/expansion-panel.common.ts

```diff
@@ -23,3 +23,5 @@ export interface IExpansionPanelEventArgs extends IBaseEventArgs {
     event: Event;
     panel: IgxExpansionPanelBase;
```

Comment:
please change `panel` to `owner`, as it is coming from `IBaseEventArgs`.
Then you can deprecate `panel`.